DigitalRain/DigitalRainGame.cs

```diff
@@ -18,33 +21,39 @@ public DigitalRainGame()
 
         protected override void Initialize()
         {
-            // TODO: Add your initialization logic here
-
+            _helloWorldPosition = new Vector2(-20, -20);
             base.Initialize();
         }
 
         protected override void LoadContent()
         {
             _spriteBatch = new SpriteBatch(GraphicsDevice);
-
-            // TODO: use this.Content to load your game content here
+            _spriteFont = Content.Load<SpriteFont>("default");
         }
 
         protected override void Update(GameTime gameTime)
         {
             if (GamePad.GetState(PlayerIndex.One).Buttons.Back == ButtonState.Pressed || Keyboard.GetState().IsKeyDown(Keys.Escape))
                 Exit();
 
-            // TODO: Add your update logic here
-
+            _helloWorldPosition = _helloWorldPosition + new Vector2(1, 1);
```

Comment:
This is overkill, but
```
//Initialization
private float speed = 0.05f
public Vector2 _velocity = new Vector2(speed)
// Not sure if 1 arg is shorthand or you need to do
new Vector2(speedX, speedY)

// Update
_helloWorldPosition += _velocity * gameTime.ElapsedGameTime.TotalMilliseconds
// By smoothing by elapsed gametime, you smooth the movement of objects regardless of framerate. Total overkill LOL
```

Reply:
We're definitely gonna use something like that when we do the falling raindrops.  But it'll also need to be discretized like I was mentioning since they don't fall "smoothly".